composer.json

```diff
@@ -26,13 +26,13 @@
     "require-dev": {
         "apigen/apigen": "^2.8",
         "pdepend/pdepend": "^2.2",
-        "phpmd/phpmd": "^2.0",
-        "phpunit/php-invoker": "^1.1",
-        "phpunit/phpunit": "^4.0",
         "sebastian/finder-facade": "^1.1",
         "sebastian/phpcpd": "^1.4",
+        "theseer/fdomdocument": "^1.6",
+        "phpmd/phpmd": "^2.0",
         "squizlabs/php_codesniffer": "^1.4",
-        "theseer/fdomdocument": "^1.6"
+        "fabpot/php-cs-fixer": "^0.3|^1.11"
```

Comment:
> Some people may have newer version of sca tools as global installations and the result will be different from others (and different from the ci builds).

That is why we check them on CI too ;)
And, if they have older - then they should update, but if they have newer - then it's great they will fix things that we didn't saw with old version of tools.

> Removed build.xml :)

great !